test/browser_spec.rb

```diff
@@ -952,6 +952,12 @@
     end
   end
 
+  it 'detects Google Structured Data Testing Tool as a bot' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.